builder/actions/setup_cross_ci_helpers.py

```diff
@@ -20,6 +20,8 @@ def create_windows_cert_store(env, certificate_env, location_env):
         return
     pfx_cert_path = env.shell.getenv(certificate_env)
 
+    env.shell.exec("powershell.exe", new_cert_folder_arguments, check=True)
```

Comment:
What's going on here?

Reply:
That's embarrassing... I forgot to remove testing code. Nice catch!! Thanks!!